modules/local/bigmag/main.nf

```diff
@@ -0,0 +1,36 @@
+process BIGMAG {
+
+    conda "conda-forge::pandas=1.4.3"
+    container "${workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container
+        ? 'https://depot.galaxyproject.org/singularity/pandas:1.4.3'
+        : 'biocontainers/pandas:1.4.3'}"
+
+    input:
+    path summary
+    path gunc_sum
+
+    output:
+    path "bigmag_summary.tsv", emit: bigmag_summary
+    path "versions.yml"   , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def summary  = summary.sort().size() > 0 ? "--summary ${summary}" : ""
+    def gunc_summary  = gunc_sum.sort().size() > 0 ? "--gunc_summary ${gunc_sum}" : ""
+    """
+    bigmag_summary.py \
+        ${args} \
+        ${summary} \
+        ${gunc_summary} \
```

Comment:
I'm wondering now why we don't have GUNC already in the `bin_summary.tsv` table by defult 🤔 I think it owuld make sense to have it anyway...